rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2135,6 +2135,12 @@ namespace rsx
 				m_flattener.force_disable();
 			}
 
+			if (async_flip_requested & flip_request::native_ui)
```

Comment:
This is a bit hacky - what happens when using Qt dialogs? There is quite a lot more work that needs to go into this. I kinda feel like this should also be commented out as a note, but if it doesn't break anything I guess it can stay for now.

Reply:
The most harm it does is spamming EINVAL in sys_rsx_context_attribute in case cellGcm isnt inited, will take care of it.